crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/space.rs

```diff
@@ -1045,7 +1046,8 @@ impl Space {
         let i_s_c = i_s.clone();
         tokio::task::spawn(async move {
             loop {
-                tokio::time::sleep(std::time::Duration::from_secs(5 * 60)).await;
+                // tokio::time::sleep(std::time::Duration::from_secs(5 * 60)).await;
+                tokio::time::sleep(std::time::Duration::from_secs(5)).await;
```

Comment:
^^ Is this an accident from debugging? I don't think we want to be updating the agent info every 5 seconds...